src/controllers/AuthLocalController.js

```diff
@@ -104,26 +173,122 @@ export default class AuthLocalController {
     const token = req.body.token || req.params.token;
     const { passwordOne, passwordTwo } = req.body;
     if (passwordOne !== passwordTwo) {
-      return res.status(status.BAD_REQUEST).send({ message: 'Passwords are not matching' });
+      return res.status(status.BAD_REQUEST).json({ errors: 'Passwords are not matching' });
     }
 
     if (!req.body.passwordOne || !req.body.passwordTwo) {
-      return res.status(status.BAD_REQUEST).send({ message: 'the password can not be empty' });
+      return res.status(status.BAD_REQUEST).json({ errors: 'the password can not be empty' });
     }
 
     const isPasswordValid = validate.password(passwordOne, 'required');
     const isPasswordValidTwo = validate.password(passwordTwo, 'required');
 
     if (isPasswordValid.length || isPasswordValidTwo.length) {
-      return res.status(status.BAD_REQUEST).send({ message: isPasswordValid[0] });
+      return res.status(status.BAD_REQUEST).json({ message: isPasswordValid[0] });
     }
     const { email } = helper.token.decode(token);
-    const updatedUser = await User.update(
-      { password: helper.password.hash(passwordOne) },
-      { email }
-    );
-    return Object.keys(updatedUser).length
-      ? res.status(status.OK).send({ message: 'Success! your password has been changed.' })
-      : res.status(status.NOT_MODIFIED).send({ message: 'Password not updated' });
+    const isUpdated = await User.update({ password: helper.password.hash(passwordOne) }, { email });
+    return isUpdated
+      ? res
+        .status(status.OK)
+        .json({ isUpdated, message: 'Success! your password has been changed.' })
+      : res.status(status.NOT_MODIFIED).json({ errors: 'Password not updated' });
+  }
+
+  // follow user
+  /**
+   * @description function to create user follows
+   * @param {object} req request from user
+   * @param {object} res server response
+   * @returns {object} true
+   */
+  static async follow(req, res) {
```

Comment:
why is the follow functionality under auth controller.

Reply:
ok I am going to migrate it to user controller

Reply:
resolved, know the follow's functionality is located to the user file